sdk/storage/Microsoft.Azure.WebJobs.Extensions.Storage.Common/src/Shared/Queues/QueueTriggerMetrics.cs

```diff
@@ -9,7 +9,7 @@ namespace Microsoft.Azure.WebJobs.Extensions.Storage.Common.Listeners
     /// <summary>
     /// Metrics type representing the status of an Azure Queue.
     /// </summary>
-    internal class QueueTriggerMetrics : ScaleMetrics
+    public class QueueTriggerMetrics : ScaleMetrics
```

Comment:
Make it internal, we will discuss making it public after the TBS change is merged.

Reply:
done